src/AppRouter.js

```diff
@@ -17,37 +20,34 @@ const AppRouter = () => (
       path="/:topic"
       render={({ history, match }) => {
         if (topicKeys.includes(match.params.topic)) {
+          const activeTopic = topics.find(t => t.key === match.params.topic);
+          const { elements } = activeTopic;
+          let menuChildren = null;
+
+          switch (match.params.topic) {
+            case 'ch.sbb.handicap': {
```

Comment:
Would be nice to make AppRouter independent from configuration details like menus specific to certain topics. But currently I don't have a good idea how to do this.

Reply:
I agree. Couldn't we define the menuChildren and popups conf in the topic config directly?

Reply:
I also agree but I don't have a solution.
Can we define react components in the config file?
However, this problem still persists. I would leave it like this until we find a better solution. Do you agree?

Reply:
👍